lib/utils/telemetry/generatePayload.js

```diff
@@ -105,6 +105,7 @@ module.exports = async ({
   serviceDir,
   configuration,
   serverless,
+  interactiveContext,
```

Comment:
I'd from a start make it command agnostic. So maybe let's simply accept `commandUsage`, which should be passed as is.